package.json

```diff
@@ -34,8 +34,9 @@
     "lib": "./lib"
   },
   "dependencies": {
-    "libxmljs": "*",
-    "qs": "^5.2.0"
+    "qs": "*",
```

Comment:
The version of "qs" needs to be locked down because the latest ones only work in node 4.x.x and above. You can use `"qs": "^5.2.0"`. This is the reason Travis builds are failing for Node 0.10 and 0.12.

Btw, it probably makes sense to do the same with "q". Running `npm install q --save` sets the following in the package.json `"q": "^1.4.1"`  so you can use that.